### ISSUE TITLE
CRF and Byte to Span models not converging during training

### ISSUE THREAD

**Issue by Akshay0799** [davidandym/multilingual-NER#16]
I was trying to replicate the performance of the CRF and Byte to Span models from the paper. 

But when I train these two models with help of the training scripts from the repo, they don't seem to converge and the loss values are big. I wanted to know if there is something that I am missing or if the default hyper parameter settings given in the repo are the correct ones ?
